services/reproducible-central/reproducible-central.service.js

```diff
@@ -0,0 +1,91 @@
+import Joi from 'joi'
+import { BaseJsonService, pathParams } from '../index.js'
+
+const schema = Joi.object({
+  color: Joi.string().required(),
+  message: Joi.string().required(),
+}).required()
+
+export default class ReproducibleCentral extends BaseJsonService {
+  static category = 'dependencies'
+
+  static route = {
+    base: 'reproducible-central/a',
+    pattern: ':groupId/:artifactId/:version',
+  }
+
+  static openApi = {
+    '/reproducible-central/a/{groupId}/{artifactId}/{version}': {
+      get: {
+        summary: 'Reproducible Central',
+        parameters: pathParams(
+          {
+            name: 'groupId',
+            example: 'org.apache.maven',
+          },
+          {
+            name: 'artifactId',
+            example: 'maven-core',
+          },
+          {
+            name: 'version',
+            example: '3.9.9',
+          },
+        ),
+      },
+    },
+  }
+
+  static defaultBadgeData = {
+    label: 'Reproducible Builds',
+    labelColor: '1e5b96',
```

Comment:
Can we remove this and stick with the standard palette. If people want to customise it, they can but it should use the standard colour by default.

Reply:
https://reproducible-builds.org/ has a specific color
if you really feel bad, we can go to simple blue, I think like "reproducible build" the color would not block people from understanding

Reply:
For all badges, the label (left side of the badge) should use the default #555
You can always override it with a `?labelColor=1e5b96` URL param when you embed it if you like

Reply:
https://reproducible-builds.org/ has a specific color
if you really feel bad, we can go to simple blue, I think like "reproducible build" the color would not block people from understanding

Reply:
if you wish: not helping the message to the user, because this is all about https://reproducible-builds.org/
but not the most important, dropping that configuration